_data/patcher-landing-page-tabs-terraform-upgrade.yml

```diff
@@ -0,0 +1,7 @@
+- id: pr
+  title: PR
+  content: https://i.imgur.com/XDAl1Ew.png
+
+- id: patch
+  title: Patch
+  content: https://i.imgur.com/mxEkeHw.png
```

Comment:
NIT: instead of an image, for code, it's more legible to include an actual code snippet in `<pre>` and `<code>` tags. See [here](https://gruntwork.io/migrate-from-heroku-to-aws#step-2-configure-ci-cd) for an example.

Reply:
I'm struggling getting this to work, as currently the tabbed divs support either images only or code tabs only, but not a mix of the two. Since it's a 'NIT' I'll leave this as is for now.